cli/admin/src/test/java/io/pravega/cli/admin/AbstractAdminCommandTest.java

```diff
@@ -44,6 +44,7 @@ public static void setUp() throws Exception {
         pravegaProperties.setProperty("pravegaservice.zk.connect.uri", SETUP_UTILS.getZkTestServer().getConnectString());
         pravegaProperties.setProperty("pravegaservice.container.count", "4");
         pravegaProperties.setProperty("pravegaservice.admin.gateway.port", String.valueOf(SETUP_UTILS.getAdminPort()));
+        pravegaProperties.setProperty("pravegaservice.clusterName", "pravega-cluster");
```

Comment:
NIT: Since we are using `SETUP_UTILS` already for fetching the values, can we not use the same pattern here as well instead of hardcoding a String value? A final constant in `SETUP_UTILS` can be created and used here.

Reply:
@shshashwat The existing values pulled up from SetupUtils are return values of methods in that class and not constants defined there.